src/community/features-templating/features-templating-core/src/main/java/org/geoserver/featurestemplating/validation/AbstractTemplateValidator.java

```diff
@@ -74,7 +74,9 @@ private boolean validateExpressions(
                 if (sb.getSource() != null && sb.getStrSource() != null) {
                     String typeName =
                             sb.getStrSource().substring(sb.getStrSource().indexOf(":") + 1);
-                    if (!getTypeName().contains(typeName)) {
+                    boolean skipValidation =
```

Comment:
Why should we skip validation for a top level feature template?

Reply:
the feature name and the featureType differs in most of the cases... Moreover AttributeExpressionImpl is unable to evaluate to FeatureType when the path starts or is constituted by the featureTypeName even if is exact.
I've added a check by  hand of the matching of the name and when failing will eventually log a warn. But the validation cannot fail for the toplevel source because the featureType name, for what I know, might vary from the top level feature name in unpredictable way in app-schema